package.json

```diff
@@ -76,6 +76,8 @@
   },
   "peerDependencies": {
     "vue-toastification": "^2.0.0-rc.5",
-    "@marshallswain/vuedraggable": "^4.2.0"
+    "@marshallswain/vuedraggable": "^4.2.0",
+    "@pinia/nuxt": "^0.11.0",
+    "pinia": "^3.0.2"
```

Comment:
have you tried with the basic nuxt `useState` ? better to not require pinia unless it's absolutely necessary

Reply:
Yes, useState works within the same component context and not between routeMiddleware and components